modules/local/cellranger/mkvdjref.nf

```diff
@@ -0,0 +1,37 @@
+process CELLRANGER_MKVDJREF {
```

Comment:
This as local module is not needed as you already included the nf-core module, right?

Reply:
yes exactly. I created this for testing as the nf-core module was not built to work with all possible inputs. 
Now that we changed this in the nf-core module, we can remove the local one 👍